openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceProfilerPipeline.json

```diff
@@ -35,12 +37,14 @@
       "default": true
     },
     "profileSample": {
-      "description": "Percentage of data used to execute the profiler for the whole workflow. This percentage will be applied to all tables in the workflow. Represented in the range (0, 100].",
+      "description": "Percentage of data we want to execute the profiler and tests on. Represented in the range (0, 100).",
       "type": "number",
-      "exclusiveMinimum": 0,
       "maximum": 100,
       "default": null
```

Comment:
You'll need to update that. The maximum should not be `100` otherwise people won't be able to sample more than 100 ROWS